operator/v1/types_network.go

```diff
@@ -372,6 +372,9 @@ type OVNKubernetesConfig struct {
 	IPsecConfig *IPsecConfig `json:"ipsecConfig,omitempty"`
 	// policyAuditConfig is the configuration for network policy audit events. If unset,
 	// reported defaults are used.
+	// This config is replaced during a strategic merge patch
+	// TODO(astoycos): once https://github.com/kubernetes-sigs/controller-tools/issues/622
```

Comment:
this ends up in the docs, probably not what you want.

Reply:
removed